src/components/book.js

```diff
@@ -40,7 +40,11 @@ function Book({ title, author, id }) {
       <div className="chapter">
         <p className="label">CURRENT CHAPTER</p>
         <p className="current">Chapter3: A lesson learned</p>
-        <button type="button" className="delete">
+        <button
+          onClick={() => dispatch(RemoveBook(id))}
+          type="button"
+          className="delete"
+        >
```

Comment:
- [ ] When I delete a book, your implementation only updates the redux store but doesn't remove the book via the API. Please interact with the API 🙏